pymedphys/cli/labs/quickcheck.py

```diff
@@ -0,0 +1,34 @@
+"""Export csv data from QuickCheck measurements.
+"""
+
+from pymedphys.labs.quickcheck import export_cli
+
+
+def quickcheck_cli(subparsers):
+    quickcheck_parser = subparsers.add_parser(
+        "quickcheck",
+        help="A toolbox to retreive measurement data from PTW Quickcheck and write it to a csv file.",
+    )
+    quickcheck_subparsers = quickcheck_parser.add_subparsers(dest="quickcheck")
+
+    export_quickcheck(quickcheck_subparsers)
+
+    return quickcheck_parser
+
+
+def export_quickcheck(quickcheck_subparsers):
+    parser = quickcheck_subparsers.add_parser(
+        "to-csv", help="Export Quickcheck data to .csv"
+    )
+
+    parser.add_argument(
+        "ip", type=str, help="IP address or host name of Quickcheck device"
+    )
+
+    parser.add_argument("csv_path", type=str, help="file path for csv file.")
+
+    parser.add_argument(
+        "-v", "--verbose", action="store_true", help="Flag to output debug information."
```

Comment:
There's actually already `--verbose` and `--debug` built into the top level CLI. These are called by running, for example:

`pymedphys --verbose labs quickcheck to-csv ip.address.goes.here /output/path/here.csv`

Probably no need to double up here. :)

Reply:
See the following code for the implementation of this top level logging settings:

https://github.com/pymedphys/pymedphys/blob/32fc4275251659ceef68bdb49aab7da6245696d5/pymedphys/cli/main.py#L49-L66

https://github.com/pymedphys/pymedphys/blob/32fc4275251659ceef68bdb49aab7da6245696d5/pymedphys/cli/main.py#L74-L75